ProgramowanieBot/Modules/ApplicationCommands/SlashCommands/ResolveCommand.cs

```diff
@@ -25,30 +26,15 @@ public class ResolveCommand(IServiceProvider serviceProvider, IOptions<Configura
         User? helper2 = null)
     {
         var configuration = options.Value;
-
-        var channel = Context.Channel;
-        var channelId = channel.Id;
+        var channelId = Context.Channel.Id;
 
         await using (var context = serviceProvider.GetRequiredService<DataContext>())
             if (await context.Posts.AnyAsync(p => p.PostId == channelId && p.IsResolved))
                 throw new(configuration.Interaction.PostAlreadyResolvedResponse);
 
-        await Context.Client.Rest.SendMessageAsync(configuration.Interaction.PostResolvedNotificationChannelId, $"**{string.Format(configuration.Interaction.PostResolvedNotificationMessage, channel)}**");
+        await PostsHelper.SendPostResolveMessagesAsync(channelId, Context.User.Id, helper.Id, helper2?.Id, Context.Client.Rest, configuration);
 
-        var isHelper2 = helper2 != null && helper != helper2;
-        var user = Context.User;
-        return InteractionCallback.Message(new()
-        {
-            Content = $"**{configuration.Emojis.Success} {(isHelper2 ? string.Format(configuration.Interaction.WaitingForApprovalWith2HelpersResponse, helper, helper2) : string.Format(configuration.Interaction.WaitingForApprovalResponse, helper))}**",
-            Components =
-            [
-                new ActionRowProperties(
-                [
-                    new ActionButtonProperties($"approve:{helper.Id}:{helper != user}:{(isHelper2 ? helper2!.Id : null)}:{(isHelper2 ? helper2 != user : null)}", configuration.Interaction.ApproveButtonLabel, ButtonStyle.Success),
-                ]),
-            ],
-            AllowedMentions = AllowedMentionsProperties.None,
-        });
+        return InteractionCallback.DeferredModifyMessage;
```

Comment:
The callback is only for component interactions

Reply:
Fixed with caab97cd5b13b852c61712d4b6e72c453128807c